### DISCUSSION TITLE
0.29.1

### DISCUSSION THREAD

**Discussion by AmrDeveloper** [AmrDeveloper/GQL#126] [Announcements]
- Implement Dynamic type system.
- Implement new Dynamic values system.
- Improve the output printer implementation.
- Change the analysis format to be similar to popular engines.
- Implement `benchmark` function.
- Implement Bitwise XOR operator for integers.
- Implement Contained By expression using `<@` operator.
- Speedup order by statement by using pre eval map.
- Support multi lines and unicode in `LIKE` and `REGEX` expressions.
- Improve error message for invalid column name.
- Improve safety check for std or aggregation signature.
- Migrate to Gix `0.67.0`.
- Integrate with LineEditor.

<hr /><em>This discussion was created from the release <a href='https://github.com/AmrDeveloper/GQL/releases/tag/0.29.1'>0.29.1</a>.</em>

**Comment by jorgeaponte** [comment]
Hey there, 
Thank you for these improvements.
I downloaded this new version and it looks like something is broken related with the operators handling:

![image](https://github.com/user-attachments/assets/91028a2f-9375-42c8-b7b4-0e86f88528cb)

**Comment by AmrDeveloper** [comment]
@jorgeaponte 

Thank you for reporting, fixed in this commit https://github.com/AmrDeveloper/GQL/commit/96e2e2dc2c36ff9ad5d1b3316b3448d363980883

Will check all other ops and create release

Thank you
